docs/smartcontracts.md

```diff
@@ -58,33 +63,7 @@ You can view this contract and deploy it to a testnet using the [Ligo WebIDE][2]
 ```
 { parameter (or (int %decrement) (int %increment)) ;
   storage int ;
-  code { DUP ;
-         CDR ;
-         DIP { DUP } ;
-         SWAP ;
-         CAR ;
-         IF_LEFT
-           { DIP { DUP } ;
-             SWAP ;
-             DIP { DUP } ;
-             PAIR ;
-             DUP ;
-             CAR ;
-             DIP { DUP ; CDR } ;
-             SUB ;
-             DIP { DROP 2 } }
-           { DIP { DUP } ;
-             SWAP ;
-             DIP { DUP } ;
-             PAIR ;
-             DUP ;
-             CAR ;
-             DIP { DUP ; CDR } ;
-             ADD ;
-             DIP { DROP 2 } } ;
-         NIL operation ;
-         PAIR ;
-         DIP { DROP 2 } } }
+  code { UNPAIR ; IF_LEFT { SWAP ; SUB } { ADD } ; NIL operation ; PAIR } }
```

Comment:
is this supposed to represent this contract? if so it needs to be updated since this one is different than the others

Reply:
fixed.
```
{ parameter (or (or (int %decrement) (int %increment)) (unit %reset)) ;
  storage int ;
  code { UNPAIR ;
         IF_LEFT { IF_LEFT { SWAP ; SUB } { ADD } } { DROP 2 ; PUSH int 0 } ;
         NIL operation ;
         PAIR } }
```